### PR TITLE
Add more terminals with color support

### PR DESCRIPTION
**Pull request by ii14**

### CODE REVIEW

File: src/colorprint.cc [hunk 1]
```diff
@@ -168,7 +168,8 @@ bool IsColorTerminal() {
       "xterm",         "xterm-color",     "xterm-256color",
       "screen",        "screen-256color", "tmux",
       "tmux-256color", "rxvt-unicode",    "rxvt-unicode-256color",
-      "linux",         "cygwin",
+      "linux",         "cygwin",          "xterm-kitty",
```
Comment (LebedevRI) [inline src/colorprint.cc:171]: Please update the URL to point at https://github.com/google/googletest/blob/v1.13.0/googletest/src/gtest.cc#L3225-L3259

File: src/colorprint.cc [hunk 2]
```diff
@@ -168,7 +168,8 @@ bool IsColorTerminal() {
...
+      "alacritty",
```
Comment (LebedevRI) [inline src/colorprint.cc:172]: Reference for `alacritty` supporting color?

Comment (ii14) [reply src/colorprint.cc:172]: Under `CSI m`: https://github.com/alacritty/alacritty/blob/edf4df66c9afcb9593eaeaaf34d5c80c66925adc/docs/escape_support.md#csi-control-sequence-introducer---esc-
terminfo: https://github.com/alacritty/alacritty/blob/edf4df66c9afcb9593eaeaaf34d5c80c66925adc/extra/alacritty.info

**Comment by ii14** [comment]
Looks like CLA requires a Google account. Is it really necessary for such a tiny change?

**Comment by dmah42** [comment]
https://cla.developers.google.com/about

i'm afraid we do need CLAs for any contributions, and our CLA does require a Google account.

**Comment by ii14** [comment]
Maybe I'll figure this out some other day then. In the meantime, if anyone wants to, feel free to apply this patch under your own name as the author.

**Comment by ii14** [comment]
Superseded by #1621

**Pull request closed without merging**